helm-charts/splunk-otel-collector/values.yaml

```diff
@@ -162,10 +162,19 @@ extraAttributes:
   # Labels that will be collected from k8s pods (in case they are set)
   # and added  as extra attributes to the telemetry in the following format:
   # k8s.pod.labels.<label_name>: <label_value>
-  podLabels:
-    - app
-    # - k8s-app
-    # - release
+  # Labels that will be collected from k8s namespaces (in case they are set)
+  # and added  as extra attributes to the telemetry in the following format:
+  # k8s.namespace.labels.<label_name>: <label_value>
+  fromLabels:
+    - key: app
+
+  # Annotations that will be collected from k8s pods (in case they are set)
+  # and added as extra attributes to the telemetry in the following format:
+  # k8s.pod.annotations.<annotation_name>: <annotation_value>
+  # Annotations that will be collected from k8s namespaces (in case they are set)
+  # and added as extra attributes to the telemetry in the following format:
+  # k8s.namespace.annotations.<annotation_name>: <annotation_value>
```

Comment:
Please add few of examples here to cover the following use cases:

- How to collect namespace attributes/labels
- How to collect all labels with `.*` 
- How to customize attribute that will be added to logs as the result